ci_framework/roles/tempest/tasks/main.yml

```diff
@@ -56,6 +56,7 @@
     volume:
       - "{{ cifmw_tempest_artifacts_basedir }}/:/var/lib/tempest/external_files:Z"
     detach: false
+    dns: "{{ cifmw_tempest_dns_servers }}"
```

Comment:
we can see this being passed here

https://review.rdoproject.org/zuul/build/03fec18dcb014e0b84720314881c33bf/log/job-output.txt#5278

2023-10-10 08:25:25.007832 | controller | TASK [tempest : Run tempest name=tempest, image={{ cifmw_tempest_image }}:{{ cifmw_tempest_image_tag }}, state=started, auto_remove={{ cifmw_tempest_remove_container | default(false) }}, network=host, volume=['{{ cifmw_tempest_artifacts_basedir }}/:/var/lib/tempest/external_files:Z'], detach=False, dns={{ cifmw_tempest_dns_servers }}, env={'CONCURRENCY': '{{ cifmw_tempest_concurrency | default(omit) }}'}] ***